### PR TITLE
LPS-144839 add test to LPS-135678 and LPS-135679

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
- LPS-144839 add test with the mimeType when returning folder content
- LPS-144839 add test to check DLFolderUtil new method
- LPS-144839 add test to check th return the folder in the results of the search
- LPS-144839 add test to check if the folder is no contained on the group do not return values

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFolderServiceTest.java [hunk 1]
```diff
@@ -91,6 +93,184 @@ public void setUp() throws Exception {
...
+			 i++) {
+
+			FileEntry expectedFileEntry = expectedFileEntries.get(i);
+			DLFileEntry actualFileEntry =
+				(DLFileEntry)actualFoldersAndFileEntriesAndFileShortcuts.get(i);
+
+			Assert.assertEquals(
+				expectedFileEntry.getFileEntryId(),
+				actualFileEntry.getFileEntryId());
+		}
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFolderServiceTest.java:184]: Here you already know the list size is 1, so why not get rid of the for loop? Also, you must use a different suffix FileEntry and DLFileEntry variables (e.g. actualDLFileEntry vs actualFileEntry).

File: modules/apps/document-library/document-library-web/src/test/java/com/liferay/document/library/web/internal/util/DLFolderUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+			RandomTestUtil.randomLong());
+	}
+
+	@Test(expected = NoSuchFolderException.class)
+	public void testValidateDepotFolderNotConnected() throws PortalException {
+		PowerMockito.mockStatic(GroupLocalServiceUtil.class);
+
+		long depotGroupId = RandomTestUtil.randomLong();
+
+		Group depot = _getDepotGroup(depotGroupId);
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/test/java/com/liferay/document/library/web/internal/util/DLFolderUtilTest.java:89]: Rename to `depotGroup`.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFolderServiceTest.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFolderServiceTest.java]: Thanks, I will add a commit renaming the variables of the file.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**